.testcaferc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "src": [
+    "browser-tests/loginView.ts", "browser-tests/registrationView.ts", "browser-tests/membershipView.ts"
```

Comment:
I have two big question marks with this approach

1. `src` setting makes no mention of being a tool for controlling execution order: https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#src
2. With TestCafe, and tests in general, it's recommended that test cases don't depend on each other.

That's why I'm hesitant to accept this change. We discussed this in more detail on Slack and didn't find a clear winning formula. The best possible alternative was one Santtu came up with and it's to use multiple test accounts that are in different states of their "lifecycle". This removes the dependency, but increases the maintenance load. I'd cautiously vote for this approach, because it allows us to avoid doing 1 or 2.

Reply:
I'll go with the multiple profile approach, after sleeping on this it makes much more sense.